modules/microshift-disconnected-host-procedure.adoc

```diff
@@ -0,0 +1,115 @@
+
+.Prerequisites
+* RHEL 9 or newer.
+* {microshift-short} 4.14 or newer.
+* Access to the host CLI.
+* A valid IP address chosen to avoid both internal and potential future external IP conflicts when running {microshift-short}.
+* {microshift-short} networking settings are set to defaults.
+
+.Procedure
+
+. Add a fake IP address to the loopback interface by running the following command:
++
+[source,terminal]
+----
+$ IP="10.44.0.1" <1>
+$ sudo nmcli con add type loopback con-name stable-microshift ifname lo ip4 ${IP}/32
+----
+<1> The fake IP address used in this example is “10.44.0.1”.
++
+[NOTE]
+====
+Any valid IP works, as long as it avoids both internal {microshift-short} and potential future external IP conflicts. This can be any subnet that does not collide with the {microshift-short} node subnet or is be accessed by other services on the device.
+====
+
+. Configure the DNS interface to use the local name server by setting modifying the settings to ignore automatic DNS and reset it to the local name server:
++
+.. Bypass the automatic DNS by running the following command:
++
+[source,terminal]
+----
+$ sudo nmcli conn modify stable-microshift ipv4.ignore-auto-dns yes
+----
++
+.. Point the DNS interface to use the local name server:
++
+[source,terminal]
+----
+$ sudo nmcli conn modify stable-microshift ipv4.dns "10.44.1.1"
+----
+
+. Get the hostname of the device by running the following command:
++
+[source,terminal]
+----
+$ NAME="$(hostnamectl hostname)"
+----
+
+. Add an entry for the hostname of the node in the `/etc/hosts` file by running the following command:
++
+[source,terminal]
+----
+$ echo "$IP $NAME" | sudo tee -a /etc/hosts >/dev/null
+----
+
+. Update the {microshift-short} configuration file by adding the following YAML snippet to `/etc/microshift/config.yaml`:
++
+[source,terminal]
+----
+sudo tee /etc/microshift/config.yaml > /dev/null <<EOF
+node:
+  hostnameOverride: hostnameOverride: $(echo $NAME)
+  nodeIP: $(echo $IP)
+EOF
+----
+
+. {microshift-short} is now ready to use the loopback device for cluster communications. Finish preparing the device for offline use.
+
+.. If the device currently has a NIC attached, disconnect the device from the network.
+.. Shut down the device and disconnect the NIC.
+.. Restart the device for the offline configuration to take effect.
+
+. Restart the {microshift-short} host to apply the configuration changes by running the following command:
++
+[source,terminal]
+----
+$ sudo systemctl reboot <1>
+----
+<1> This step restarts the cluster. Wait for the greenboot health check to report the system healthy before implementing verification.
+
+.Verification
+
+At this point, network access to the {microshift-short} host has been severed. Use the host CLI to verify that the cluster has started in a stable state.
```

Comment:
We should mention that the state of the cluster can only be verified if the user has access to the device terminal.  Something like:

```suggestion
At this point, network access to the {microshift-short} host has been severed. If you have access to the host terminal, you can use the host CLI to verify that the cluster has started in a stable state.
```